mithril-stm/src/participant/signer.rs

```diff
@@ -1,7 +1,8 @@
 use crate::bls_multi_signature::{Signature, SigningKey, VerificationKey};
 use crate::eligibility_check::ev_lt_phi;
 use crate::key_reg::ClosedKeyReg;
-use crate::stm::{Stake, StmParameters, StmSig};
+use crate::single_signature::StmSig;
+use crate::stm::{Stake, StmParameters};
 use blake2::digest::{Digest, FixedOutput};
```

Comment:
It's more readable to dispatch the imports this way: standard library, line break, external dependencies, line break and then crate.
```suggestion
use blake2::digest::{Digest, FixedOutput};

use crate::bls_multi_signature::{Signature, SigningKey, VerificationKey};
use crate::eligibility_check::ev_lt_phi;
use crate::key_reg::ClosedKeyReg;
use crate::single_signature::StmSig;
use crate::stm::{Stake, StmParameters};
```